### DISCUSSION TITLE
Views render issue.

### DISCUSSION THREAD

**Discussion by erenergul** [filamentphp/filament#2209] [General]
Hello , Everything is okay on my system and working good. 

But when i open simple category table page from filament admin. It rendered 274 page. I will put screenshot.I just want to know.  Is this normal ? Because it takes long to open simple table with rendering 274 page. I also check other page. 188 - 130 - 150 around like that. Is there any option to optimize it ? 

<img width="1680" alt="Ekran Resmi 2022-04-23 21 38 27" src="https://user-images.githubusercontent.com/23069775/164938874-1e11f679-c629-4df6-a5c7-3ccebb341b6f.png">

**Comment by Z3d0X** [comment]
its not 274 "pages", it is blade view files. Several tiny components together make the page in Filament. If you look closely at the  debug bar you will notice these components, `header-cell`, `search-input` `bulk-actions-trigger` etc.

I checked a couple of filament projects with debug bar it seem to load somewhere between ~175, even all the way upto 300, with no noticeable performance issues. If you are experiencing performance issues it could be elsewhere?

**Reply by danharrin** [reply]
4MB memory use, 106ms response, that's very good compared to other projects!

**Reply by erenergul** [reply]
Thank you for your answers. Some of them take longer to load and they re  still less than 100ms.  What is demo project using on production ? Any cache ? redis ? As i believe it takes longer on my project because of category images because its only difference between mine and demo project.

**Reply by danharrin** [reply]
No cache in production really, just Cloudflare. $5 DO droplet & Laravel Forge :D
